test/unit/range/view/view_istreambuf_test.cpp

```diff
@@ -47,17 +47,16 @@ TEST(view_istreambuf, basic)
     is.clear();
     is.seekg(0, std::ios::beg);
     auto v3 = view::istreambuf(is) | view::char_to<dna5> | view::complement;
-    std::vector<dna5> out = v3;
     std::vector<dna5> comp{"TGCATATATATANTATATANAATNNNTATATT"_dna5};
-    EXPECT_TRUE(std::ranges::equal(out, comp));
+    EXPECT_TRUE(std::ranges::equal(v3, comp));
```

Comment:
This helped?!

Reply:
Yup... Because now there is no explicit conversion to `std::vector<dna5>` again.